01.fizzbuzz/fizzbuzz.js

```diff
@@ -0,0 +1,17 @@
+#!/usr/bin/env node
+
+function fizzbuzz() {
+  for(let i = 1; i <= 20; i++ ){
+    if (i % 15 == 0) {
+      console.log("FizzBuzz") 
```

Comment:
余分な空白が含まれているのと、行末にセミコロンがないです。

```suggestion
      console.log("FizzBuzz");
```